taskotron_python_versions/unversioned_shebangs.py

```diff
@@ -52,8 +52,12 @@ def get_scripts_summary(package):
 
     for shebang in FORBIDDEN_SHEBANGS:
         if shebang_to_require(shebang) in package.require_names:
+            log.debug('Package {} requires {}'.format(
+                package.filename, shebang_to_require(shebang)))
```

Comment:
Since the require is bytes, this will be weird on Python 3:

```pycon
>>> 'Package {} requires {}'.format('foo', b'/usr/bin/env')
"Package foo requires b'/usr/bin/env'"
```